index.html

```diff
@@ -0,0 +1,38 @@
+<html>
+
+<head>
+    <title>
+        Master Dex
+    </title>
+    <link href="index.css" rel="stylesheet" type="text/css" />
+    <meta name="viewport" content="width=device-width, intial-scale=1" />
+    <meta charset="utf-8" />
+</head>
+
+</html>
+
+<body>
+    <div id="background"></div>
+    <div class="background-top">
+        <div class="pinkBall-01"></div>
+        <div class="pinkBall-02"></div>
+    </div>
+    <div class="master-text">
+        <p>Master Dex</p>
+    </div>
+    <div class="background-center">
+
+        <div class="center-line">
+            <div class="center-circle"></div>
+            <div class="search-bar">
+                <div class="input"></div>
```

Comment:
I don't see any inputs on your landing page.  If this is not necessary yet, please do not include it in the PR.

Reply:
I am using the input div class. I did notice that once the page was in an iphone window viewport that certain objects were moved around so in the next ticket i'll adjust that.